src/components/TaskListItem.js

```diff
@@ -0,0 +1,180 @@
+import DataStore from '../modules/DataStore.js';
+import Task from '../modules/Task.js';
+
+export const refreshTaskList = () => {
+  const tasks = document.querySelectorAll('.task');
+  tasks?.forEach((prevTask) => {
+    prevTask.parentNode.removeChild(prevTask);
+  });
+
+  DataStore.tasks = DataStore.updateTaskIndex();
+  DataStore.tasks.forEach((task) => {
+    // eslint-disable-next-line
+    renderTaskItem(task);
+  });
+};
+
+export const isTaskCompleted = () => {
+  const completedTasks = document.querySelectorAll('input[name="isCompleted"]');
+  completedTasks.forEach((btnIsCompleted) => {
+    btnIsCompleted.addEventListener('click', () => {
+      if (btnIsCompleted.checked) {
+        btnIsCompleted.checked = true;
```

Comment:
- [ ] **Required** The value of `isCompleted`  still doesn't change to true when clicked, please revisit your approach from when the item object(description, completed, index) is created to the point where you try to make the change between states.

![state false is unchanged](https://user-images.githubusercontent.com/27709832/151581971-846f1593-21af-4637-84f1-3f51434af965.gif)